lib/requests/diaryFoods.js

```diff
@@ -18,6 +18,14 @@ const getMeals = () => {
     .catch(error => console.error({ error })
 )}
 
+const deleteFoodFromMeal = (mealName, foodId) => {
+  const mealInfo = { 'breakfast': '1', 'snack': '2', 'lunch': '3', 'dinner': '4' }
+  let mealId = mealInfo[`${mealName}`]
+  fetch(`https://ivmh-qs-api.herokuapp.com/api/v1/meals/${mealId}/foods/${foodId}`, {
```

Comment:
Interesting approach of using the data attribute and accessing a substring  - using it to access a meal id for a fetch request.